modules/memberships/application/use-cases/create-checkout-session.ts

```
@@ -403,6 +408,47 @@ interface CreateAndPersistCheckoutSessionParams {
+		return params.run();
+	}
+
+	const requestContext = createRequestContext({
+		service: "memberships.create_checkout_session",
+		requestId: params.requestId,
+		startTime: params.clock.now().getTime(),
+		clock: params.clock,
+	});
+
+	return runWithContext(requestContext, params.run);
+}
```

Biome (2.3.14)

[error] This async function lacks an await expression.

Remove this async modifier, or add an await expression in the function.

Async functions without await expressions may not need to be declared async.

(lint/suspicious/useAwait)